src/structural/Organ.cpp

```diff
@@ -850,7 +850,11 @@ double Organ::getLatGrowthDelay(int ot_lat, int st_lat, double dt) const //overr
 	double growthDelay; //store necessary variables to define lateral growth delay
 	int delayDefinition = std::static_pointer_cast<const SeedRandomParameter>(getOrganism()->getOrganRandomParameter(Organism::ot_seed,0))->delayDefinition;
 
-	assert(std::isfinite(delayDefinition)&&(delayDefinition>=0)); 
+#ifdef _MSC_VER
+	assert(delayDefinition >= 0); // msvc does not support std::isfinite for types that are not float or double
+#else
+	assert(std::isfinite(delayDefinition) && (delayDefinition >= 0));
+#endif
 	
```

Comment:
actually, can t we simply convert delayDefinition to float? (and same for the other time we use isfinite)

Reply:
Float conversion after it was originally an int will not possibly yield an infinite value iirc, because all possible bit combinations refer to a number in integer values, and thus always a correct and finite float value will be printed.